src/java-interop/java-interop-util.h

```diff
@@ -1,6 +1,8 @@
 #ifndef __JAVA_INTEROP_UTIL_H__
 #define __JAVA_INTEROP_UTIL_H__
 
+#include <stdlib.h>
```

Comment:
If this header is to be consumed only by C++ code the include should be `<cstdlib>` instead

Reply:
The reason this needed to be added was because of the `exit()` call in `_assert_valid_pointer()`.  If we use `<cstdlib>`, we'll need to update that to `std::exit()`, which should be "fine"…

Reply:
Including `<cstdlib>` still allows us to use just `exit()`, the header merely does some additional C++ magic to the definitions found in the C `stdlib.h`.

Reply:
Done.